src/bsp_types/mappings/build_target.rs

```diff
@@ -66,7 +66,7 @@ fn tags_and_capabilities_from_cargo_kind(
     (tags, capabilities)
 }
 
-pub fn new_bsp_build_target(
+pub fn bsp_build_target_from_cargo_target(
```

Comment:
Zmienilbym na to cargo_target_to_bsp_build_target

Reply:
to chyba subiektywne mocno